.github/workflows/version.yml

```diff
@@ -0,0 +1,26 @@
+name: Upgrade Version in config.json for evcc-io/hassio-addon
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  upgrade-version-config-json:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pocket-apps/action-update-version@v1
```

Comment:
Committed das die Datei auch wieder? Ist eigentlich nicht nötig und wäre schöner ohne. Hier würde ja ein Platzhalter 0.0.0 reichen.

Bzgl. Version: aktuell haben wir nur 0.100, nur bei bugfix 0.100.1. Macht das Probleme?

Reply:
nein, das "version-regexp: '\d+.\d+.\d+'" such nur die version, geschrieben wird das ergebniss immer hinter "version:" in "".
Somit wird egal was du als releast 0.100 oder auch 0.100.599 oder 0.100.1 korrelt geschrieben (das habe ich getestet).